### ISSUE TITLE
The plugin doesn't work on windows

### ISSUE THREAD

**Issue by myartings** [charlesfleche/usdStl#1]
Just follow the tutorial , build and install the plugin , it says " Cannot determine file format for @d:/Dev/GitHub/usdStl/src/testenv/cube.stl:SDF_FORMAT_ARGS:target=usd@". The detail is below, I Don't have any idea about this issue.  My USD version is 23.02, built in release.  I Need help.
![image](https://github.com/charlesfleche/usdStl/assets/4220108/aa69674a-1a3c-40ec-9cd8-b070b131f78f)

**Comment by myartings** [comment]
I set TF_DEBUG environment variable to "*" prior to launching usdview. Here is the log.
[stl_log.txt](https://github.com/charlesfleche/usdStl/files/11865131/stl_log.txt)

**Comment by charlesfleche** [comment]
The log seems to indicate that the plugin, both the json and the dll, are properly logged. Maybe the build system requires something else for Windows ? I'm not sure, and I don't have a Windows machine to quickly try myself.
Have you tried in both Debug and Release mode ?

**Comment by myartings** [comment]
Yes, I finally build the debug version of USD, and build the plugin in Debug mode, it works.  But I don't know why release version doesn't work.
